main.go

```diff
@@ -12,25 +12,27 @@ import (
 	"time"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/prometheus/common/log"
 )
 
-const applicationName = "opentsdb_exporter"
+const (
+	applicationName = "opentsdb_exporter"
+	metricsRoute    = "/metrics"
+	probeRoute      = "/opentsdb"
```

Comment:
Maybe it should be `/probe` ? I'm not sure if `/opentsdb` is the best name for it.